bench-tps/src/main.rs

```diff
@@ -121,7 +123,7 @@ fn main() {
             &client,
             Some(*faucet_addr),
             &id,
-            *tx_count,
+            keypair_count,
```

Comment:
Is this right? It needs to fund all sets of keypairs?

Reply:
No, it's wrong. Only half need to be funded, thanks!

Reply:
Well, it actually looks like the current behavior (before my change) is to fund all keypairs as well. Not sure when that happened but I think it can be fixed in a follow up issue, if that's cool!

Reply:
Is it? Because it creates tx_count * 2 keypairs, then it funds only tx_count of them.

Reply:
Then it does the ping-pong between those two sets.

Reply:
Hmm seems like that's the intended behaviour but looks like since https://github.com/solana-labs/solana/pull/1436 it funds all of the accounts. Am I missing something?

Reply:
Ah, I suppose so. ok, we can keep that behavior for now.